src/rust/kanto-tui/kantui_conf.toml

```diff
@@ -15,12 +22,20 @@ remove_kbd_key = "r"
 logs_btn_name = "[L]ogs"
 logs_kbd_key = "l"
 
-quit_btn_name = "[Q]uit"
-quit_kbd_key = "q"
+quit_btn_name = "^[Q]uit"
```

Comment:
Having to use `Ctrl-Q` (instead of simply `q`) is a bit annoying in Visual Studio, where I run leda inside of a terminal inside of qemu. VSCode is stealing the Ctrl-Q.

Workaround is to exit kantui using Ctrl-C in that case. Or enabling the passthru of hotkeys in VSCode.